docs/pages/reference/cli.mdx

```diff
@@ -2034,3 +2028,221 @@ $ tsh ls --search=staging,mac
+
+The following command highlights how to set permissions with tbot through Linux groups, using the user and group `jenkins:jenkins`.
+If running `tbot` as the Linux user `root`, use the following invocation of
+`tbot init` to initialize the short-lived certificate directory
+`/opt/machine-id` with owner `jenkins:jenkins`.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --owner=jenkins:jenkins
+```
+
+**Example using POSIX ACLs.**
+
+If running `tbot` as the Linux user `teleport`, use the following invocation of
+`tbot init` to initialize the short-lived certificate directory
+`/opt/machine-id` with owner `teleport:teleport` but allow `jenkins` to read
+from `/opt/machine-id`.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --reader-user=jenkins
+```
+
+
+### tbot db
+
+Connects to databases using native clients and queries database information. This is best used for testing and validation purposes;
+most users will likely prefer to connect their own databases to a local proxy using `tbot proxy db`.
+
+Note that tsh must be installed to make use of this command.
+
+#### Flags
+
+| Flag                | Description                                                                                              |
+|---------------------|----------------------------------------------------------------------------------------------------------|
+| `-d/--debug`        | Enable verbose logging to stderr.                                                                        |
+| `-c/--config`       | Path to a configuration file.                                                                            |
+| `--destination-dir` | Path to the Machine ID destination dir that should be used for authentication. Required.                 |
+| `--proxy`           | The `host:port` of the Teleport Proxy Service to use to access resources. Required.                      |
+| `--cluster`         | The name of the cluster on which resources should be accessed. Extracted from the bot identity if unset. |
+
+All other flags and arguments are passed directly to `tsh db ...`, along
+with authentication parameters to use the Machine ID identity to skip `tsh`'s
+login steps.
+
+Note that certain CLI parameters, for example `--help`, may be captured by
+`tbot` even if intended to be passed to the wrapped `tsh`. A `--` argument can
+be used to ensure all following arguments are passed to `tsh` and ignored by
+`tbot`.
+
+Additionally, be aware of the following limitations of `tbot db`:
+ - `tbot db connect` requires a `tbot db login` for certain database types,
+   like MySQL, so that additional connection parameters can be written to a
+   local configuration file.
+ - `tbot db env` is not fully supported.
+
+### tbot proxy
+
+Allows for access to teleport resources via a proxy using various protocols. Requires running a local ALPN/SNI proxy to access the resources if using TLS.
```

Comment:
```suggestion
Allows for access to Teleport resources via a proxy using various protocols. Requires running a local ALPN/SNI proxy to access the resources if using TLS.
```

Reply:
It's difficult for me to tell what this command does from this paragraph. 

- What does "via a proxy" mean? What kind of proxy is this? How would a user know whether a given proxy is relevant to this command?

- How do you know whether you have the required local ALPN/SNI proxy running? Also, `tsh proxy db` starts the local ALPN/SNI proxy for you—does this command do that as well?